antwar.config.js

```diff
@@ -89,6 +89,8 @@ module.exports = {
           false,
           /^\.\/.*\.md$/
         );
+      }, {
+        'external-config': 'javascript-alternatives'
```

Comment:
This must be called `external-configs` (with a trailing `s`) conforming with the former filename. Sorry for my typo!